orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -97,3 +97,20 @@ def _create_iobinding(io_binding, inputs, model, device):
 
     for value_info in model.graph.output:
         io_binding.bind_output(value_info.name, device.type, device_id=get_device_index(device))
+
+class ModuleMetadata():
+    """Encapsulates modules and allows easy access as required"""
+
+    def __init__(self, original_module, flattened_module):
+        self._original_module = original_module
+        self._flattened_module = flattened_module
+
+    def original_module(self):
+        """Returns the original PyTorch module"""
+
+        return self._original_module
+
+    def flattened_module(self):
+        """Returns the flattened input and output PyTorch module"""
+
+        return self._flattened_module
```

Comment:
Nit: Since this is an internal utility class and the names are pretty descriptive, it would be fine to use public members directly, e.g.

```suggestion
    def __init__(self, original_module, flattened_module):
        self.original_module = original_module
        self.flattened_module = flattened_module
```